packages/sampler/stories/default/ViewManager.js

```diff
@@ -0,0 +1,135 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select} from '@enact/storybook-utils/addons/knobs';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import Group from '@enact/ui/Group';
+import Item from '@enact/ui/Item';
+import Layout, {Cell} from '@enact/ui/Layout';
+import Scroller from '@enact/ui/Scroller';
+import {arrange, SlideArranger, SlideBottomArranger, SlideLeftArranger, SlideRightArranger, SlideTopArranger, ViewManager, ViewManagerBase} from '@enact/ui/ViewManager';
+import {useCallback, useState} from 'react';
+
+import css from './ViewManager.module.less';
+
+const ViewManagerConfig = mergeComponentMetadata('ViewManager', ViewManagerBase, ViewManager);
+
+ViewManagerConfig.defaultProps = {
+	duration: 300,
+	enteringDelay: 0
+};
+
+const prop = {
+	arranger: ['SlideBottomArranger', 'SlideLeftArranger', 'SlideRightArranger', 'SlideTopArranger', 'SlideArranger', 'CustomArranger (FadeAndSlideArranger)'],
+	direction: ['bottom', 'left', 'right', 'top']
+};
+
+const arrangers = {
+	'SlideBottomArranger': SlideBottomArranger,
+	'SlideLeftArranger': SlideLeftArranger,
+	'SlideRightArranger': SlideRightArranger,
+	'SlideTopArranger': SlideTopArranger
+};
```

Comment:
```suggestion
const arrangers = {
	SlideBottomArranger,
	SlideLeftArranger,
	SlideRightArranger,
	SlideTopArranger
};
```
But, it still not enough.
If i were you I will use only one variable.
it duplicated with prop.arranger

```

const prop = {
	arranger: {
		SlideBottomArranger,
		SlideLeftArranger,
		SlideRightArranger,
		SlideTopArranger,
		SlideArranger: null,
		CustomArranger: null
	},
	direction: ['bottom', 'left', 'right', 'top']
};


	const arrangerType = select('arranger', Object.keys(prop.arranger), ViewManagerConfig, 'SlideBottomArranger');
	
		arranger = prop.arranger[arrangerType];
```

Reply:
That's what I should know. Thank you.